tools/js-sdk-release-tools/src/utils/git.ts

```diff
@@ -45,7 +45,7 @@ export async function getLastCommitId(repository: string) {
     try {
         commitId = execSync(`git --git-dir=${path.join(repository, '.git')} log --format=%H -n 1`, {encoding: "utf8"});
     } catch (e) {
-        logger.info(`cannot get commit id from ${repository}`);
+        logger.info(`Failed to get commit id from '${repository}'.`);
```

Comment:
Why does this message log as info level? Should we tell user the commit id is incorrect or it is a retriable error?